### PR TITLE
Add API endpoint to fetch an entity

### PR DESCRIPTION
**Pull request by fguillot**

### CODE REVIEW

File: almanach/adapters/api_route_v1.py [hunk 1]
```diff
@@ -289,6 +289,15 @@ def entity_exists(entity_id):
     return response
 
 
+@api.route("/entity/<entity_id>", methods=["GET"])
+@authenticated
+@to_json
+def get_entity(entity_id):
+    if not controller.entity_exists(entity_id=entity_id):
```
Comment (mbelang) [inline almanach/adapters/api_route_v1.py]: I think this logic should be part of the controller, not the API.

File: almanach/adapters/database_adapter.py [hunk 2]
```diff
@@ -91,6 +91,11 @@ def list_entities(self, project_id, start, end, entity_type=None):
         return [build_entity_from_dict(entity) for entity in entities]
 
     @database
+    def get_all_entities_by_id(self, entity_id):
+        entities = self.db.entity.find({"entity_id": entity_id}, {"_id": 0})
```
Comment (mbelang) [inline almanach/adapters/database_adapter.py:95]: If the instance has been resize multiple times, we will return all streams of the instance. It is the requested behaviour or should we return the last occurrence?

Comment (fguillot) [inline almanach/adapters/database_adapter.py:95]: I assume that we want the history of a given entity (all streams)?

**Comment by Marx314** [comment]
:+1:

**Comment by mbelang** [comment]
:+1:

**Pull request merged**